libs/datalayout/src/main/java/com/linkedin/openhouse/datalayout/strategy/DataLayoutStrategy.java

```diff
@@ -11,6 +11,7 @@
 @Data
 @Builder
 public class DataLayoutStrategy {
+  private final long computationTimestamp;
```

Comment:
```suggestion
  private final long createTimestamp;
```

Reply:
maybe just timestamp if we don't plan to add other time attributes

Reply:
Just named it timestamp.